Makefile

```diff
@@ -39,3 +44,17 @@ format: format_jsonnet
 .PHONY: format_jsonnet
 format_jsonnet: $(JSONNET_FILES)
 	$(JSONNET_DOCKER) $(JSONNETFMT_ARGS) -- $?
+
+compile:
+	$(COMMODORE_CMD)
```

Comment:
We should probably run `jb install` first?

Then you can move the `kube.libsonnet` dependency to the `jsonnetfile.json` instead of vendoring it manually.

Reply:
Where is this file? I could not find it in this repo

Reply:
You can create it with `jb init`. There's no file by default...

Reply:
How and from where would I declare the dependency from commodore? Do you have an example?

Reply:
You can just declare Jsonnet libraries: https://git.vshn.net/syn/commodore-components/synsights-metrics/-/blob/master/jsonnetfile.json

Reply:
thanks, that worked